src/server/connection.go

```diff
@@ -130,9 +130,12 @@ func newConnection(completeWorld *ecs.World, id briskconnection.ID, root *protoc
 	messageHandler.Connection = c
 	changeHandler.Connection = c
 
-	connectErr := loadBalancer.OnSimulatorConnected(completeWorld.EntityManager(), c.participant, transferManager)
-	if connectErr != nil {
-		return nil, connectErr
+	// Simulators are supplied to the load balancer
+	if connectionType == Simulator {
+		connectErr := loadBalancer.OnSimulatorConnected(completeWorld.EntityManager(), c.participant, transferManager)
+		if connectErr != nil {
+			return nil, connectErr
+		}
```

Comment:
If we are only checking for an error, it is more nice to have them on one line. Also the `connectErr` variable is only scoped in the if statement. which is great.

```suggestion
		if connectErr := loadBalancer.OnSimulatorConnected(completeWorld.EntityManager(), c.participant, transferManager); connectErr != nil {
			return nil, connectErr
		}
```